go.mod

```diff
@@ -4,8 +4,8 @@ go 1.17
 
 require (
 	github.com/confluentinc/bincover v0.1.0
-	github.com/cosmos/cosmos-sdk v0.45.3
-	github.com/cosmos/ibc-go/v3 v3.0.0
+	github.com/cosmos/cosmos-sdk v0.45.4
+	github.com/cosmos/ibc-go/v3 v3.0.0-alpha1.0.20220510152759-dcd0681d8f07
```

Comment:
I guess one can't put `ics29-beta2` as the version string?

Reply:
Correct. It's just a version tag and not a release, so, `go mod tidy` automatically replaces it with this version.